element_deeplabcut/model.py

```diff
@@ -3,17 +3,25 @@
 MIT License Copyright (c) 2022 Mackenzie Mathis
 DataJoint Schema for DeepLabCut 2.x, Supports 2D and 3D DLC via triangulation.
 """
-
 import datajoint as dj
 import os
+import csv
+import cv2
+import yaml
 import inspect
 import importlib
 import numpy as np
+import pandas as pd
 from pathlib import Path
 from datetime import datetime
-import yaml
-import cv2
+from packaging import version
 from element_interface.utils import find_full_path, find_root_directory
+from distutils.util import strtobool
+from deeplabcut import evaluate_network
+from deeplabcut import __version__ as dlc_version
+from deeplabcut.utils.auxiliaryfunctions import GetScorerName
+from deeplabcut.utils.auxiliaryfunctions import get_evaluation_folder
+from .readers import dlc_reader

```

Comment:
Why were the imports moved up? They were previously moved to within the relevant functions to avoid unnecessary imports for users who won't use all functions. `cv2` is particularly heavy and I would be in favor of keeping it in the relevant function

Reply:
By the same token, I would like to discourage the use of `pandas` for things like reading a CSV file.  The standard library does a better job and it's much more stable and lightweight.

Reply:
In this case, `pandas` was used to avoid refactoring code shared by the Mathis team. See [`get_trajectory`](https://github.com/tdincer/element-deeplabcut/blob/main/element_deeplabcut/model.py#L630-L661)

Reply:
That part of the code is not reading a csv file but creating/returning a data frame to be used in somewhere else.

Reply:
I am cautious about using pandas as a principal datatype for passing around data.  Numpy recarrays are more lightweight and more stable/established than pandas.